src/aiodynamo/errors.py

```diff
@@ -1,5 +1,10 @@
 import json
-from typing import Any, Dict
+from typing import Any, Dict, List, Optional, TypedDict
+
+
+class CancellationReason(TypedDict):
+    Code: Optional[str]
+    Message: Optional[str]
```

Comment:
to keep things consistent with how other parts of aiodynamo work, this should be a (frozen) dataclass using pythonic identifiers (no uppercase for attributes).

Reply:
Should I also convert:


```
No Errors:
    Code: None
    Message: null
```

to just be a `None` value in the array or leave it as a `CancellationReason(code="None", message=None)`?

ref: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html#API_TransactWriteItems_Errors

Reply:
using `None` for those cases seems like a good idea